### ISSUE TITLE
Missing viaRelation, viaResource, ... parameters

### ISSUE THREAD

**Issue by gmooren** [64robots/nova-fields#152]
Different relation fields are not working when using the quickCreate functionality. I noticed that the viaResource parameters are not sent along with the request for creation-fields and update-fields. 

For example the basic Nova functionality performs a request to:
`/nova-api/users/creation-fields?editing=true&editMode=create&viaResource=&viaResourceId=&viaRelationship=`

The quick create function however performs the following request:
`/nova-api/users/creation-fields`

This is the reason why relation fields are not working. I tried to make a pull request but I am not able to build a proper JS distribution package. It looks like dependencies in package.json are not working together anymore. When upgrading to laravel-mix > 4.0 the build succeeds but gives errors when using it:

```
[Vue warn]: Failed to mount component: template or render function not defined.

found in

---> <FormNovaFieldsBelongsTo>
       <Card> at resources/js/components/Card.vue
         <FormPanel> at resources/js/components/Form/Panel.vue
           <LoadingView> at resources/js/components/LoadingView.vue
             <Create> at resources/js/views/Create.vue
               <Root>
```

**Comment by gmooren** [comment]
Solved the issue in the build process by fixing the webpack version to 3.12.0. See PR for the fix.

**Comment by disitec** [comment]
Thanks @gmooren will take a look at the fix tomorrow!

**Comment by disitec** [comment]
Fix by @gmooren available in 0.16.0 🎉
